notebooks/matrix_tube_pipeline_seqcount_norm.ipynb

```diff
@@ -699,15 +890,20 @@
   },
   {
    "cell_type": "code",
-   "metadata": {},
+   "execution_count": null,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2025-03-04T00:18:23.023016Z",
+     "start_time": "2025-03-04T00:18:21.704827Z"
+    }
+   },
+   "outputs": [],
    "source": [
     "plot_plate_vals(dna_concs[np.ix_(ODD_ROWS,ODD_COLS)],\n",
```

Comment:
THis and the prior two cells look like they do the same `np.ix_` trick. Out of scope for this PR, but I wonder whether `plot_plate_vals` should do that tinkering internally?

Reply:
Yes, probably ;) Comment above :D